feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.html

```diff
@@ -12,4 +12,16 @@
       </button>
     </div>
   </ng-container>
+  <ng-template #done>
+    <ng-container *ngIf="order$ | async as order">
```

Comment:
This does not cover the checkout. In this case, we also run this page in displayOnly mode and want to show the done button. Here, the button needs to trigger a browser back.